.changeset/brown-mice-hug.md

```diff
@@ -0,0 +1,7 @@
+---
+"@wso2is/admin.users.v1": patch
```

Comment:
change the state to minor for users and console

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/8171/commits/7b8ce3f507869014664af33d012595e94cd13ad9